src/fairseq2/models/wav2vec2/_handler.py

```diff
@@ -52,7 +52,10 @@ def _convert_checkpoint(
 def convert_wav2vec2_checkpoint(
     checkpoint: dict[str, object], config: Wav2Vec2Config
 ) -> dict[str, object]:
-    state_dict = cast(MutableMapping[str, Tensor], checkpoint["model"])
+    model_key: str = "model"
```

Comment:
Nit: these three lines can be simplified to `checkpoint.get("model_key", "model")`